package.json

```diff
@@ -8,6 +8,8 @@
     "turbo": "next dev --turbo --port 4444",
     "build": "next build",
     "start": "next start -p 4444",
+    "test": "vitest",
```

Comment:
Thought about adding jest but since vitest is already installed and Next.js has official support, just went with it.